.github/styles/config/vocabularies/Seqera/accept.txt

```diff
@@ -41,8 +41,8 @@ FSx
 Dockerfile
 initContainers
 Cron
-Autoscaling
-Autoscale
+[Aa]utoscaling
+[Aa]utoscale
```

Comment:
Do we need to add things like Fusion, MultiQC, and Wave here? Especially if we are treating them as proper nouns?

Reply:
@christopher-hakkaart, probably eventually; it only triggers if a file in a given PR has a violation, so we don't need to accumulate all the possibilities before merging.

Reply:
All good 👍